pkg/subctl/cmd/cloud/cleanup/rhos.go

```diff
@@ -0,0 +1,56 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cleanup
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/submariner-io/cloud-prepare/pkg/api"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/cmd/cloud/rhos"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/cmd/utils"
+)
+
+// NewCommand returns a new cobra.Command used to prepare a cloud infrastructure.
+func newRHOSCleanupCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "rhos",
```

Comment:
Just a note (no action required in this pr): We are using generic names like rhos, aws, gcp etc but they work mainly with OCP alone. Probably we should use names to convey that its for OCP on aws/gcp/rhos etc.